modules/ROOT/pages/assembly-pipeline-resource-types.adoc

```diff
@@ -0,0 +1,30 @@
+[]
+= PipelineResources
+
+
+`PipelinesResources` in a pipeline are the set of objects that are going to be
+used as inputs to a `Task` and can be output by a `Task`.
+
+A `Task` can have multiple inputs and outputs.
+
+To define a configuration file for a `PipelineResource`, you must specify the
+following fields:
+
+*   Required:
+    **  `apiVersion`- Specifies the API version
+    **   `kind` - Specify the `PipelineResource` resource object.
```

Comment:
Do we need to say "Specify the `PipelineResource` resource object"? I would rather say, "Specify the `PipelineResource` object."